.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Run CI
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v4
+
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          cache: 'pip'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pipenv
+          pipenv install --dev
+
+      - name: Black
+        run: python -m black --check src/
+
+      - name: Pylint
+        run: python -m pylint **/*.py
```

Comment:
These are in use on https://github.com/hackgvl/slack-events-bot. Do we want to enforce the same linting rules in this repository? I gave each a shot locally, and the diff would be quite large. 

I think there would be a lot of value in having consistency across the org, though, so it may be worth it.